python-packages/sra_client/sra_client/__init__.py

```diff
@@ -2,6 +2,145 @@
+                        'maker_asset_data': '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
+                        'maker_fee': '0',
+                        'salt': '2362734632784682376287462',
+                        'sender_address': '0x0000000000000000000000000000000000000000',
+                        'taker_address': '0x0000000000000000000000000000000000000000',
+                        'taker_asset_amount': '500000000000000000000',
+                        'taker_asset_data': '0xf47261b00000000000000000000000002002d3812f58e35f0ea1ffbf80a75a38c32175fa',
+                        'taker_fee': '0'}}]}
+
+
+Get Order
+---------
+
+Get an order by hash from an SRA-compliant Relayer.
+
+>>> example_order_hash = (
+...     "0xc1c4e9a983755b4a2ff048b0c591a27"
+...     "0f437972e1ec440986770ac943a577404")
+>>> relayer_api.get_order(order_hash=example_order_hash)  # doctest: +SKIP
+{'meta_data': {},
+'order': {'exchange_address': '0x35dd2932454449b14cee11a94d3674a936d5d7b2',
+            'expiration_time_seconds': '1000000000000000000000',
+            'fee_recipient_address': '0x0000000000000000000000000000000000000000',
+            'maker_address': '0x5409ed021d9299bf6814279a6a1411a7e866a631',
+            'maker_asset_amount': '1000000000000000000',
+            'maker_asset_data': '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
+            'maker_fee': '0',
+            'salt': '2362734632784682376287462',
+            'sender_address': '0x0000000000000000000000000000000000000000',
+            'taker_address': '0x0000000000000000000000000000000000000000',
+            'taker_asset_amount': '500000000000000000000',
+            'taker_asset_data': '0xf47261b00000000000000000000000002002d3812f58e35f0ea1ffbf80a75a38c32175fa',
+            'taker_fee': '0'}},
+
+Get Asset Pair
+---------------
+
+Get available asset pairs from an SRA-compliant Relayer.
+
+>>> relayer_api.get_asset_pairs()
+{'records': [{'assetDataA': {'assetData': '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
+                             'maxAmount': '115792089237316195423570985008687907853269984665640564039457584007913129639936',
+                             'minAmount': '0',
+                             'precision': 18},
+              'assetDataB': {'assetData': '0xf47261b00000000000000000000000002002d3812f58e35f0ea1ffbf80a75a38c32175fa',
+                             'maxAmount': '115792089237316195423570985008687907853269984665640564039457584007913129639936',
+                             'minAmount': '0',
+                             'precision': 18}}]}
+
+Get Orderbook
+-------------
+
+Get the orderbook for an asset pair from an SRA-compliant Relayer.
+
+
+>>> from zero_ex.order_utils import asset_data_utils
+>>> example_base_asset_data = asset_data_utils.encode_erc20_asset_data(
+...     "0xd0a1e359811322d97991e03f863a0c30c2cf029c")
```

Comment:
Use `NETWORK_TO_ADDRESSES[NetworkId.KOVAN.ether_token`, or re-use prior variable containing that.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1734/commits/338895ef102cb7508d6b41640d7f43d057743f59